members/B000444.yaml

```diff
@@ -0,0 +1,106 @@
+bioguide: B000444
+contact_form:
+  method: post
+  action: ""
+  steps:
+    - visit: https://www.whitehouse.gov/contact/organization
+    - fill_in:
+        - name: submitted[first_name]
```

Comment:
Yeah, I was going by the specifications listed here: https://github.com/NGPVAN/contact-congress/blob/master/documentation/schema.md

"name: The name HTML attribute of the field to be filled out."